doc/recipe/preprocessor.rst

```diff
@@ -926,10 +926,20 @@ example of its usage in an ESMValTool preprocessor is:
             reference: esmf_regrid.schemes:ESMFAreaWeighted
             mdtol: 0.7
 
-.. TODO: Remove the following warning once things have settled a bit.
-.. warning::
-   Just as the mesh support in Iris itself, this new regridding package is
-   still considered experimental.
+Additionally, the use of generic schemes that take source and target grid cubes as 
+arguments is also supported. The `regrid` module will automatically pass 
+the cubes as inputs of the scheme. An example of this usage is
+the `regrid_rectilinear_to_rectilinear` scheme available in `iris-esmf-regrid`:
```

Comment:
```suggestion
the :func:`~esmf_regrid.schemes.regrid_rectilinear_to_rectilinear` scheme available in :doc:`iris-esmf-regrid:index`:
```

Not a 100% sure if this will work, but we can try